client/reader/following/main.jsx

```diff
@@ -47,7 +47,12 @@ function FollowingStream( { suggestions, ...props } ) {
 		suggestions &&
 		suggestions
 			.flatMap( ( query ) => [
-				<Suggestion suggestion={ query.text } source="following" railcar={ query.railcar } />,
+				<Suggestion
+					suggestion={ query.text }
+					source="following"
+					railcar={ query.railcar }
+					key={ query.railcar.railcar }
```

Comment:
nit: can we move `key` to the top of the list?

Reply:
Sure! Done in db2794e8ed2907cd416019e6ee5666b5154a2fd7